ProjectSidewalk/SidewalkWebpage
# Resolves 1294: Ensure that the Sign-In Modal Closes on the Audit Page

Resolves #1294 

Previously, the sign-in modal would not hide after a successful sign in on the audit page only. This is a very simple fix to get that working correctly. 

 To test:
- Try signing in on the audit page to see if it does disappear
- Sign in on any other page to ensure nothing is broken
- Try an incorrect sign in to make sure the modal doesn't close accidentally
- Play around with signing in as administrator and audit to make sure that nothing gets interfered with.

I have done all of these tests on my machine (in both Chrome and Firefox) and everything works as intended.
